### ISSUE TITLE
Batch normalisation

### ISSUE THREAD

**Issue by oubino** [oubino/cellpose#1]
During finetuning removed - as I think the running mean of our small dataset was throwing the model off

https://github.com/oubino/cellpose/blob/506c5a55bbca2cf7d06bc2de0aaa3e59bc853cd1/cellpose/__main__.py#L343

Maybe add back in
